Makefile

```
@@ -56,3 +56,24 @@ validate_all: examples/*
 	for file in $^ ; do \
 		$(MAKE) validate CONFIG=$${file} || exit 1; \
 	done
+
+.PHONY: smoke
+smoke:
+	@echo
+	@echo "+++ basic smoke test - does it start?"
+	@echo "+++ if so, make unsmoke after this"
+	@echo
+	mkdir -p tmp
+	go run ./cmd/hpsf -i ./examples/hpsfProxy.yaml -o tmp/hpsfProxy.cconfig.yaml cConfig
+	docker run --rm -d \
```

If we use `-it` instead of `-d`, the smoke test will run the collector and connect it's output directly to the terminal. We then don't need to `unsmoke` to tear down the container either as it'll be cleaned once you ctrl-c to exit from `smoke`.

```suggestion
	docker run --rm -it \
```

---

Sure that works. I'd like to run it in CI eventually but this works for local!